src/platform/site-wide/sass/merger.scss

```diff
@@ -825,8 +825,12 @@ img[data-srcset] {
   padding-right: 1.2em;
   position: relative;
 }
-.usa-alert {
-  .top-of-page {
-    margin-top: 0;
+
+.usa-alert-full-width {
+  background-color: $color-gray-lightest;
+  .usa-alert {
+    border-left: none;
+    max-width: 1000px;
+    margin: 0 auto;
```

Comment:
Would this be better off in Formation?

Reply:
This feels kind of homepage/header specific to me, not something we could really reuse easily.